packages/client-sdk-nodejs/src/cache-client.ts

```diff
@@ -68,6 +76,7 @@ export class CacheClient extends AbstractCacheClient implements ICacheClient {
   }
 
   public close() {
+    CacheClient.semaphore.purge();
```

Comment:
If someone instantiated multiple cache clients, then closing a single client would purge data from all clients, right? Below I see a comment that we want the limit to apply to all cache client instances, so we definitely want cleanup to apply selectively.